src/Cake.Common/Tools/NuGet/Pack/NuspecProcessor.cs

```diff
@@ -45,6 +62,24 @@ public FilePath Process(FilePath nuspecFilePath, NuGetPackSettings settings)
             return SaveNuspecXml(nuspecFilePath, xml);
         }
 
+        private static XmlDocument LoadEmptyNuSpec()
+        {
+            XmlDocument xml = new XmlDocument();
+            xml.LoadXml(@"<?xml version=""1.0"" encoding=""utf-8""?>
+<package xmlns:xsi=""http://www.w3.org/2001/XMLSchema-instance"" xmlns:xsd=""http://www.w3.org/2001/XMLSchema"">
+  <metadata xmlns=""http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd"">
+    <id></id>
+    <version>0.0.0</version>
+    <authors></authors>
+    <description></description>
+  </metadata>
+  <files>
+    <file src=""LICENSE"" />
```

Comment:
no, files are replaced when specified(so wontr end up in nuspec that gets packed), but will remove.